public/cli/lib/rendering/renderFrames.ts

```diff
@@ -6,7 +6,9 @@ import { EventEmitter } from 'events';
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const DataURI = require('datauri').promise;
 import { record } from './recordFrames';
-import { AnimationSettings, Timings, ProjectData } from '../../../models';
+import { AnimationSettings } from '../../../models/animationSettings.model';
```

Comment:
what is the best way to do TS imports so that we don't have to do relative "../.." all the time.  Any ideas?

Reply:
For it to work in the compiled JS the paths need to all be relative. Anything you do outside of that you must make sure that the compiled result has relative paths otherwise you will get module load errors at runtime. See this https://stackoverflow.com/questions/42356508/how-to-get-tsc-to-resolve-absolute-paths-when-importing-modules-using-baseurl. I did find this article that claims you can do it https://medium.com/@jimcraft123hd/setting-up-path-alias-in-typescript-and-tsc-build-without-error-9f1dbc0bccd2.

You can make an alias in `tsconfig.json`, e.g from SF:
```
{
  "compilerOptions": {
    "baseUrl": "./",
    "paths": {
      "xforge-common/*": ["./src/xforge-common/*"]
    }
  }
}
```
And then use it to import `import { AuthGuard } from 'xforge-common/auth.guard';`, but that only works because we pre-compile the *xforge-common* library and so I think the paths have already been flattened in the output folder.